doc/api/next_api_changes/2018-09-25-AL.rst

```diff
@@ -0,0 +1,7 @@
+Deprecations
+````````````
+
+The following API elements are deprecated:
+
+- the ``frameon`` kwarg to ``savefig`` and the ``savefig.frameon`` rcParam (use
+  ``facecolor`` instead),
```

Comment:
Can you please be a bit more specific? What values does one have to pass to `facecolor` to obtain the effect of `frameon=True/False`.

Reply:
clarified